localized_fields/value.py

```diff
@@ -228,3 +228,8 @@ def __int__(self):
             return self.default_value
 
         return int(value)
+
+    def __str__(self) -> str:
+        """Returns string representation of value"""
+
+        return str(self.get(translation.get_language()))
```

Comment:
This shouldn't be `translation.get_language()`. It might return `None`. The correct way to implement `__str__` for `LocalizedIntegerValue` is to do:

```
value = self.translate()
return str(value) if value else none
```

The `translate` method implements the same logica as the base class's `translate()` function. It grabs the value in the current language or the available fall back language.